config/routes.rb

```diff
@@ -1,3 +1,14 @@
 Rails.application.routes.draw do
+  devise_for :users
+  resources :posts do
+    resources :comments
+  end
+  resources :blogs do
+    resources :posts
+    get 'posts/show'
+    get 'posts/index'
+    get 'posts/new'
```

Comment:
Okay, I will get rid of them. I know I was having some trouble with routes, but I think this might have been a left over from when I first started, and I wanted to navigate to show, index etc on the browser.